pkg/markdown/md.go

```diff
@@ -23,14 +23,26 @@ import (
 	"github.com/yuin/goldmark/renderer/html"
 )
 
+type ifm struct {
+}
+
+// IFM is an extension that provides Istio Flavored markdown functionalities.
+var IFM = &ifm{}
+
+func (e *ifm) Extend(m goldmark.Markdown) {
+	extension.Table.Extend(m)
+	extension.Strikethrough.Extend(m)
+	extension.TaskList.Extend(m)
+}
+
 var md = goldmark.New(
-	goldmark.WithExtensions(extension.GFM),
```

Comment:
@ericvn I do this hack to avoid change in istio/api, but really think the new render is right.

current:
![image](https://user-images.githubusercontent.com/4354057/195488366-f9c93f87-fd89-45d0-b5a2-0c3ecd3d56e0.png)
the link should be wrap by an `<a></a>`

Reply:
I think GFM providing the link is better.

Reply:
updated